examples/myapp-02 build error on aya_gen::btf_types

After removing Cargo.lock, getting the following build error in `examplse/myapp-02`:

```
   Compiling xtask v0.1.0 (/Users/dmitris/dev/hack/gh/aya-rs/book/examples/myapp-02/xtask)
error[E0432]: unresolved import `aya_gen::btf_types`
 --> xtask/src/codegen.rs:1:5
  |
1 | use aya_gen::btf_types;
  |     ^^^^^^^^^^^^^^^^^^ no `btf_types` in the root

For more information about this error, try `rustc --explain E0432`.
error: could not compile `xtask` due to previous error
```

Apparently an API has changed and the examples need to be updated (`myapp-03` has the same issue).
@alessandrod provided the link to https://github.com/aya-rs/aya/pull/297 in Discord.

--- a/examples/myapp-02/xtask/src/codegen.rs
+++ b/examples/myapp-02/xtask/src/codegen.rs
@@ -1,16 +1,16 @@
-use aya_gen::btf_types;
+use aya_gen::generate::InputFile;
 use std::{
     fs::File,
     io::Write,
-    path::{Path, PathBuf},
+    path::PathBuf,
 };
 
 pub fn generate() -> Result<(), anyhow::Error> {
     let dir = PathBuf::from("myapp-ebpf/src");
     let names: Vec<&str> = vec!["ethhdr", "iphdr"];
-    let bindings = btf_types::generate(Path::new("/sys/kernel/btf/vmlinux"), &names)?;
+    let bindings = aya_gen::generate(InputFile::Btf(PathBuf::from("/sys/kernel/btf/vmlinux")), &names, &[])?;
     // Write the bindings to the $OUT_DIR/bindings.rs file.
     let mut out = File::create(dir.join("bindings.rs"))?;
-    write!(out, "{}", bindings)?;
+    write!(out, "{:?}", bindings)?;
     Ok(())
 }

--- a/examples/myapp-03/xtask/src/codegen.rs
+++ b/examples/myapp-03/xtask/src/codegen.rs
@@ -1,16 +1,16 @@
-use aya_gen::btf_types;
+use aya_gen::generate::InputFile;
 use std::{
     fs::File,
     io::Write,
-    path::{Path, PathBuf},
+    path::PathBuf,
 };
 
 pub fn generate() -> Result<(), anyhow::Error> {
     let dir = PathBuf::from("myapp-ebpf/src");
     let names: Vec<&str> = vec!["ethhdr", "iphdr"];
-    let bindings = btf_types::generate(Path::new("/sys/kernel/btf/vmlinux"), &names)?;
+    let bindings = aya_gen::generate(InputFile::Btf(PathBuf::from("/sys/kernel/btf/vmlinux")), &names, &[])?;
     // Write the bindings to the $OUT_DIR/bindings.rs file.
     let mut out = File::create(dir.join("bindings.rs"))?;
-    write!(out, "{}", bindings)?;
+    write!(out, "{:?}", bindings)?;
     Ok(())
 }

--- a/src/aya/aya-gen.md
+++ b/src/aya/aya-gen.md
@@ -34,7 +34,7 @@ FLAGS:
     -V, --version    Prints version information
 
 SUBCOMMANDS:
-    btf-types
+    generate
     help         Prints this message or the help of the given subcommand(s)
 ```
 
@@ -45,7 +45,7 @@ in `myapp` subdirectory, your eBPF part is in `myapp-ebpf`. We need to generate
 the bindings for the eBPF part, which can be done with:
 
 ```bash
-aya-gen btf-types task_struct > myapp-ebpf/src/vmlinux.rs
+aya-gen generate task_struct > myapp-ebpf/src/vmlinux.rs
 ```
 
 > 💡 **HINT: Generating for multiple types**
@@ -53,7 +53,7 @@ aya-gen btf-types task_struct > myapp-ebpf/src/vmlinux.rs
 > You can also specify multiple types to generate, for example:
 >
 > ```bash
-> aya-gen btf-types task_struct dentry > vmlinux.rs
+> aya-gen generate task_struct dentry > vmlinux.rs
 > ```
 >
 > But in the following example, we will focus only on `task_struct`.

--- a/src/programs/lsm.md
+++ b/src/programs/lsm.md
@@ -96,7 +96,7 @@ Therefore we need to generate a binding to `task_struct` with aya-gen.
 > [this section](../aya/aya-gen.md).
 
 ```bash
-aya-gen btf-types task_struct > lsm-nice-ebpf/src/vmlinux.rs
+aya-gen generate task_struct > lsm-nice-ebpf/src/vmlinux.rs
 ```
 
 Now it's time to modify the `lsm-nice-ebpf` project and write an actual program

